templates/workloads/partials/cf/envvars.yml

```diff
@@ -4,8 +4,10 @@
 Environment:
 - Name: COPILOT_APPLICATION_NAME
   Value: !Sub '${AppName}'
+{{- if not .Subscribe}}
```

Comment:
We should leave this in as an environment variable; this variable is for customers to use to construct queries to *other* services in the environment. 

For example:
```
# From inside worker service
curl db.$COPILOT_SERVICE_DISCOVERY_ENDPOINT:80/inventory/remove/item?=dogbowl&count?=2
```